### ISSUE TITLE
MD014 incorrectly warns for code blocks with only blank lines

### ISSUE THREAD

**Issue by DavidAnson** [markdownlint/markdownlint#117]
Example that demonstrates the issue:

``````
```js

```
``````

See also DavidAnson/markdownlint@26ab9a5596be6d4afe6fe5107b9553bb6b281c4a

**Comment by grantneufeld** [comment]
I just ran into this today, some years later. What will it take to fix this one? Thanks.

**Comment by jaymzh** [comment]
As I've mentioned elsewhere, we're very responsive to PRs, but I won't have dedicated time to spend working this, likely for the next 2 months.
